packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -344,11 +344,11 @@ class SlackService {
         let payloadContent = '';
 
         if (connectionCount === 0) {
-            payloadContent = `[Resolved] *${syncName}* (${syncType.toLowerCase()}) failed. Read <${getBasePublicUrl()}/${envName}/activity?activity_log_id=${originalActivityLogId}|logs>.`;
+            payloadContent = `[Resolved] *${syncName}* (${syncType.toLowerCase()}) failed. Read <${baseUrl}/${envName}/activity?activity_log_id=${originalActivityLogId}|logs>.`;
         } else {
             const count = connectionCount;
             const connection = count === 1 ? 'connection' : 'connections';
-            payloadContent = `*${syncName}* (${syncType.toLowerCase()}) is failing for ${count} ${connection}. Read <${getBasePublicUrl()}/${envName}/activity?activity_log_id=${originalActivityLogId}|logs>.`;
+            payloadContent = `*${syncName}* (${syncType.toLowerCase()}) is failing for ${count} ${connection}. Read <${baseUrl}/${envName}/activity?activity_log_id=${originalActivityLogId}|logs>.`;
```

Comment:
is this fix correct? seems like it should use `basePublicUrl`

Reply:
Thanks, updated